packages/api-v4/src/cloudpulse/types.ts

```diff
@@ -342,3 +342,8 @@ export interface DeleteAlertPayload {
   alertId: number;
   serviceType: string;
 }
+
+export interface CloudPulseAlertsPayload {
+  system: Alert['id'][];
+  user: Alert['id'][];
```

Comment:
```suggestion
  system: number[];
  user: number[];
```

it is a number array only, why to complicate